packages/lb-annotation/src/utils/tool/MarkerUtils.ts

```diff
@@ -10,4 +21,66 @@ export default class MarkerUtils {
   static getMarkerShowText(marker: string | undefined, markerList: IInputList[] = []) {
     return AttributeUtils.getAttributeShowText(marker, markerList);
   }
+
+  /**
+   * v3.2.0 仅标点工具 列表标注 根据配置生成辅助线数组,start和end为IMarkerList[attr]
+   * eg:[{key:"类别1",value:"类别1",target:["class-lm"]},{key:"类别lm",value:"class-lm",target:["类别1"]}] => [{start:"类别1",end:"类别lm"}]
+   */
+  static getAuxiliaryLineByMarkerList = (markerList: IMarkerList[], attr: 'key' | 'value'): IAuxiliaryLine[] => {
+    const queue = cloneDeep(markerList);
+
+    const auxiliaryLines: IAuxiliaryLine[] = [];
+
+    let currentNode = queue.shift();
+    while (currentNode) {
+      // eslint-disable-next-line no-loop-func
+      currentNode.target?.forEach((element) => {
+        const targetMarkerListItem = queue.find((i) => i.value === element);
+        if (targetMarkerListItem && currentNode?.[attr]) {
+          auxiliaryLines.push({
+            start: currentNode[attr],
+            end: targetMarkerListItem[attr],
+          });
+        }
+      });
+
+      currentNode = queue.shift();
+    }
+
+    return auxiliaryLines;
+  };
+
+  /**
+   * 根据辅助线和点数据生成辅助线首尾坐标
+   */
+  static getAuxiliaryLineCoord = (auxiliaryLines: IAuxiliaryLine[], pointList: IPointUnit[]): IAuxiliaryLineCoord[] => {
+    const result: IAuxiliaryLineCoord[] = [];
+
+    auxiliaryLines.forEach((line) => {
+      const { start, end } = line;
+      const startPoint = pointList.find((item) => item.label === start);
```

Comment:
Why is label used? I think it is more correct to use attribute

Reply:
markerList use `label`
attributeList use `attribute`

Reply:
Yes, you are right!